src/components/Sprint.vue

```diff
@@ -81,20 +105,22 @@
     <td>
       <span>{{ sprint.name }}</span>
     </td>
-    <td>
+    <td v-if="!editMode">
       <span>{{ sprint.startDate }}</span>
     </td>
-    <td>
+    <td v-if="!editMode">
       <span>{{ sprint.endDate }}</span>
     </td>
-    <td>
+    <td v-if="!editMode">
```

Comment:
I get what you mean about having the `editMode` being everywhere. I think it's ok for now though. We can have a rethink about this later and try come up with a component solution maybe (not sure if that work)

Reply:
Absolutely. I thought of having a component for editing but thought it was overkill for MVP. Going forward we can have a think about alternatives.